libtenzir/builtins/commands/exec.cpp

```diff
@@ -95,7 +95,7 @@ auto add_implicit_source_and_sink(pipeline pipe) -> caf::expected<pipeline> {
 auto format_metric(const metric& metric) {
   auto result = std::string{};
   auto it = std::back_inserter(result);
-  const auto locale = std::locale("en_US.UTF-8");
+  const auto& locale = std::locale::classic();
```

Comment:
I think you may as well delete the entire line and remove the usage of the locale in this function. The classic locale has no thousands separators, and that was the only thing this was used for.

Reply:
I think we should rather keep it to avoid confusion in case the global one ever gets imbued with something else.

Reply:
If you remove the `:L` from the format strings then the locale isn't use in any ways, which I think would be the better way to go about this rather than overriding the locale.

Reply:
That's also fine with me.